src/ocm/api/InfraEnvService.ts

```diff
@@ -0,0 +1,11 @@
+import { AxiosPromise } from 'axios';
+import { InfraEnv, InfraEnvCreateParams, InfraEnvList } from '../../common/api/types';
+import { client } from './axiosClient';
+
+export const registerInfraEnv = (params: InfraEnvCreateParams): AxiosPromise<InfraEnv> =>
```

Comment:
I'd stick with using CRUD based names for API functions (`createInfraEnv`, `deleteInfraEnv`)

Reply:
I'm trying to be consistent with the same terminology the BE is using.
[This doc](https://docs.google.com/spreadsheets/d/1GRs2eJSM8Fy7zGO80cNKyJxQByOicNnJP42R2FfEfZ4/edit#gid=1989175040) is my guide